Nickvision.Aura/Aura.cs

```diff
@@ -15,9 +16,14 @@ public class Aura
     public AppInfo AppInfo { get; init; }
 
     /// <summary>
-    /// Occurs when there are start arguments
+    /// Dictionary of configuration files that were set
     /// </summary>
-    public event EventHandler<Dictionary<string, string>>? StartArgumentsReceived;
+    public Dictionary<string, object> ConfigFiles;
```

Comment:
Maybe we make an IConfiguration interface in which every Configuration object implements that interface that way the dictionary can be `<string, IConfiguration>` instead of <string, object>`. And inside Save and Load config we can do `T where IConfiguration`...if you agree i'll implement it

Reply:
Sure, I agree